### PR TITLE
created a basic random AI which makes a random move from the set of a…

### PR DESCRIPTION
**Pull request by chastise**
…ll legal moves

### CODE REVIEW

File: players/random_player.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    def make_random_legal_move(self, game_state):
+        chosen_move = random.choice(self.construct_all_legal_moves(game_state))
+        try:
+            assert chosen_move.is_playable(game_state)
+        except AssertionError:
+            print "Chosen move was unplayable: {move} ;".format(move=chosen_move)
+            print(game_state.board)
+            print(game_state.player_hands)
+            print(game_state.player_id)
+            chosen_move.is_playable(game_state)
```
Comment (nlettner) [inline players/random_player.py]: Was this supposed to be a print statement?

File: players/random_player.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        return chosen_move
+
+    def construct_all_legal_moves(self, game_state):
```
Comment (nlettner) [inline players/random_player.py]: Not immensely important, but this function has a *lot* of nesting.

File: players/random_player.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
+from engine.player import Player
+from engine.move import Move
+
+import random
+
+
+class RandomPlayer(Player):
+    """A real live human player"""
```
Comment (nlettner) [inline players/random_player.py]: Documentation jokes are the best jokes.

**Review by nlettner — APPROVED** [review]
Everything looks ready to merge. Comments are mostly superficial.

File: players/random_player.py (hunk 1 above)
Comment (chastise) [reply players/random_player.py]: debugging. tested a couple thousand times so it's probably fine. It's possible there are some EoG edge cases I haven't considered, but I can't think of any at the moment.

File: players/random_player.py (hunk 2 above)
Comment (chastise) [reply players/random_player.py]: Probably leaving this for now. There aren't very many not-clunky ways to break it up, at best I'd split off information-getting but it's still pretty argument-passing-heavy and a generic-ized version feels over-engineered.

**Pull request merged**